bundles/admin/admin-announcements/publisher/AnnouncementsTool.js

```diff
@@ -65,7 +65,10 @@ Oskari.clazz.define('Oskari.admin.bundle.admin-announcements.publisher.Announcem
                     toolPluginAnnouncementsConf.config.announcements.forEach(announcement => {
                         const filteredAnnouncement = me.announcements.filter(ann => ann.id === announcement);
                         if (me.isAnnouncementValid(filteredAnnouncement[0])) {
```

Comment:
Using find and renaming the config array item would make this more readable:
```suggestion
                    toolPluginAnnouncementsConf.config.announcements.forEach(announcementId => {
                        const announcement = me.announcements.find(ann => ann.id === announcementId );
                        if (me.isAnnouncementValid(announcement )) {
```